include/deal.II/matrix_free/operators.h

```diff
@@ -1041,15 +1039,13 @@ namespace MatrixFreeOperators
           const VectorizedArrayType *               in_array,
           VectorizedArrayType *                     out_array) const
   {
-    internal::CellwiseInverseMassMatrixImpl<dim,
-                                            fe_degree,
-                                            n_components,
-                                            VectorizedArrayType>::
-      apply(fe_eval.get_shape_info().data.front().inverse_shape_values_eo,
-            inverse_coefficients,
-            n_actual_components,
-            in_array,
-            out_array);
+    internal::
+      CellwiseInverseMassMatrixImpl<dim, fe_degree, VectorizedArrayType>::apply(
+        fe_eval.get_shape_info().data.front().inverse_shape_values_eo,
```

Comment:
Does it make sense to make `n_components` the first argument here too, for consistency?

Reply:
Yes, I will do that.